src/i18n/locales/zh-CN

```diff
@@ -0,0 +1,375 @@
+    },
+    "requires_license": {
+      "all": "请注意!运行该核心需要取得许可授权",
+      "jotego": "The JOTEGO betas require a JTBETA file from the Patreon in the sponsors section below. \nThe jtbeta.zip should be placed at the root of your SD card then it'll be found & copied to the right place during the \"Required Files\" process."
+    //Myself couldn't fully understand the sentence above
+    },
+    "not_in_inventory": "核心{coreName}尚未加入清单",
+    "platform": {
+      "edit": "编辑",
+      "wikipedia": "维基百科"
+    },
+    "replaced": "此核心已经由<1>{replacementCore}</1>核心替代",
+    "platforms": "平台",
+    "please_wait": "请稍后，检查文件中...",
+    "release_date": "发布日期",
+    "releases": {
+      "release_history": "版本发布历史",
+      "show_previous": "显示之前的版本发布历史信息"
+    },
+    "required_files": "必要文件",
+    "required_files_count": "{count} / {total} 文件",
+    "settings": "配置",
+    "sponsor": "Sponsor",
+    "supports": "Supports",
+    "supports_items": {
+      "cartridges": "卡带",
+      "dock": "底座", //do you mean the 99.99 USD Analogue Dock? If it is, the translation is correct
+      "dock_dac": "DAC解码器", // are u refering this item? https://www.analogue.co/dac
+      "save_states_and_sleep": "即时存档/睡眠"
+    },
+    "url": "网页地址",
+    "version": "版本"
+  },
+  "core_info_required_files": {
+    "close": "关闭",
+    "download_all": "下载全部",
+    "no_link_tip": "请进入配置页面查询必要文件的更多选项",
+    "skip_alternates_label": "跳过次要文件",
+    "title": "必要文件",
+    "file_status": {
+      "wrong": "可更新",
+      "downloadable": "可下载",
+      "not_in_archive": "未入库",
+      "ok": "已下载",
+      "unknown": "未知",
+      "at_root": "已存在于根目录"
+    }
+  },
+  "cores": {
+    "controls": {
+      "category": "类型",
+      "refresh": "刷新",
+      "search": "搜索",
+      "updatable": "可更新"
+    },
+    "install_tip": "您也可以通过拖拽ZIP压缩包到窗口内的方式安装核心或其他内容",
+    "installed": "已安装({count})",
+    "not_installed": "可安装({count})"
+  },
+  "error": {
+    "report": "您可以通过GitHub报告错误信息（含错误信息和错误发生的描述性文字）",
+    "retry": "重试",
+    "title": "错误"
+  },
+  "fetch": {
+    "controls": {
+      "add_fetch_item": "Add new fetch",
+      "refresh": "刷新"
+    },
+    "copying": "正在复制文件...",
+    "fetch_button": "Fetch",
+    "file_count": "{count} / {total, plural, one {# 文件} other {# 文件}}",
+    "loading": "载入中……",
+    "new": {
+      "add_extension": "添加",
+      "add_fetch": "Add fetch",
+      "archive_name": "Archive name: <i>https://archive.org/details/[this bit here]</i>",
+      "archive_tip": "This won't work for achives where the files are zipped, and you should set which extensions you want to avoid downloading the metadata files.",
+      "close": "关闭",
+      "destination": "目标目录",
+      "extensions": "限制文件扩展名(类型):",
+      "origin": "源目录",
+      "title": "Add new Fetch"
+    },
+    "remove_button": "移除",
+    "remove_confirm": "This'll remove the fetch configuration, but not the files. Are you sure?",
+    "types": {
+      "archive_org": "在线文件", //refering to online(local&internet) source, correct?
```

Comment:
It's specifically for files on the https://archive.org/ website

Reply:
Updated accordingly.